src/app/cluster/cluster-details/node-list/node-list.component.html

```diff
@@ -228,6 +228,11 @@
               <div value>{{element.spec.cloud.openstack.image}}</div>
             </km-property>
 
+            <km-property *ngIf="element.spec.cloud.openstack && element.spec.cloud.openstack.availabilityZone">
```

Comment:
In the HTML templates you can use `?.` for null checks, i.e. `element.spec.cloud.openstack?.availabilityZone` should do the same as above. 

https://angular.io/guide/template-syntax#safe-navigation-operator